readthedocsext/theme/templates/profiles/partials/github_oauth_revoke_list.html

```diff
@@ -43,7 +43,7 @@
   {% else %}
     <a href="{{ old_application_link }}"
        target="_blank"
-       class="ui button"
+       class="ui button disabled"
```

Comment:
Nitpick, but classes are easier to maintain when kept in semantic order. In some cases explicit order is required. If you look at any example in FUI docs, you'll see they all read like sentences in english -- adjectives/modifiers before the noun.

```suggestion
       class="ui disabled button"
```